packages/core/src/bodl/bodl-service.ts

```diff
@@ -1,4 +1,16 @@
+import { BodlEventsPayload } from './bodl-window';
+
 export default interface BodlService {
     checkoutBegin(): void;
     orderPurchased(): void;
+    stepCompleted(step?: string): void;
```

Comment:
All the methods here seem to be taking an optional parameter? Is that by design?

Reply:
Yes, it is by design, because for some cases this options can be not set and received as undefined. We handle this undefined cases in code if it needed.